src/EnergyPlus/DataConvergParams.hh

```diff
@@ -101,6 +101,10 @@ namespace DataConvergParams {
 
 		// Default Constructor
 		HVACNodeConvergLogStruct() :
+			NodeNum( 0 ),
+			NotConvergedHumRate( false ),
```

Comment:
Yes I guess they can be deleted.  I don't recall the details as to why they are there but I agree they can be deleted.